src/main/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryService.java

```diff
@@ -324,7 +324,7 @@ public String exportQueueToLogString(TelemetryEvent event) {
     private TelemetryService instance;
     private String payload;
     private String payloadLogStr;
-    private static final int TIMEOUT = 3000; // 3 second timeout limit
```

Comment:
@sfc-gh-azhan Do we have a specific reason to set TIMEOUT as 3s before?

Reply:
I think the reasoning behind 3 seconds was just to not have the JDBC tests take too long. However, since there's not that many of these timeout tests,  I think it's okay to increase it to 5 seconds.

Reply:
Sounds good, let's try!